### PR TITLE
Add require block to example from readme

### PR DESCRIPTION
**Pull request by Marton6**

### CODE REVIEW

**Review by SimonBarendse — COMMENTED** [review]
This only works with Terraform 0.13, right?

Should we add instructions for both Terraform 0.13 and Terraform 0.12 and earlier?

**Comment by Marton6** [comment]
> This only works with Terraform 0.13, right?
> 
> Should we add instructions for both Terraform 0.13 and Terraform 0.12 and earlier?

That's right I'll add a separate example for each.

File: README.md [hunk 1]
```diff
@@ -23,6 +23,55 @@ It is officially supported and actively maintained by SecretHub, but community c
...
+  engine_version       = "5.7"
+  instance_class       = "db.t2.micro"
+  name                 = "mydb"
+  username             = secrethub_secret.db_username.value
+  password             = secrethub_secret.db_password.value
+  parameter_group_name = "default.mysql5.7"
+}
+```
+
+Have a look at the [reference docs](https://secrethub.io/docs/reference/terraform/) for more information on the supported resources and data sources.
```
Comment (SimonBarendse) [inline README.md:69]: Should we update these to point to the registry page? (also the one below)

**Review by SimonBarendse — COMMENTED** [review]
:+1: Shall we delay merging this until the provider is added to the registry and the example in this PR works?

File: README.md (hunk 1 above)
Comment (Marton6) [reply README.md:69]: Oh good point!

Comment (Marton6) [reply README.md]: Fixed

**Review by SimonBarendse — APPROVED** [review]

**Pull request merged**